### PR TITLE
Redirect from /devs to the community landing page.

### PR DESCRIPTION
**Pull request by kantai**
Redirect to https://community.blockstack.org/developers-getting-started from `blockstack.org/devs/`

### CODE REVIEW

File: app/js/Routes.js [hunk 1]
```diff
@@ -101,6 +101,11 @@ export default (
 
       <Route path="/press" component={PressPage} />
 
+      <Route path="/devs" component={() => {
```
Comment (larrysalibra) [inline app/js/Routes.js]: we should do this at the server level instead of in javascript. will push a commit

**Comment by kantai** [comment]
That approach is way better. Thanks @larrysalibra

**Comment by larrysalibra** [comment]
@kantai changed this to do redirect on server.
test as follows:

```
Vanquish:blockstack.org larry$ curl -I https://deploy-preview-542--blockstack.netlify.com/devs
HTTP/2 301 
cache-control: public, max-age=0, must-revalidate
content-length: 0
content-type: text/plain; charset=utf-8
date: Sat, 31 Mar 2018 02:51:39 GMT
location: https://community.blockstack.org/developers-getting-started
x-robots-tag: noindex
age: 0
server: Netlify

Vanquish:blockstack.org larry$
```

**Comment by kantai** [comment]
That works great for me, merging.

**Pull request merged**